docs/src/pages/landing-page/Index.vue

```diff
@@ -226,13 +213,45 @@ export default defineComponent({
   border-bottom: 1px solid $lp-light;
 }
 
-.support {
-  height: 100vh;
-}
-
 .social-channels-call-to-action {
   // TODO: reduce image height (from top), and remove this negative margin
   margin-top: -30%;
 }
 
+.why-quasar-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```

Comment:
For this I had difficulties manipulating it using first-child once I used min-max, if I used something like repeat(3, 1fr) then I could use first-child to manipulate the last child. But then that started requiring me to use media queries which I was trying to avoid in the first place when I used minmax. I did some research and saw some solutions. But opted out of those solutions after seeing how lengthy and hacky they were. I preferred using the html row/col solution, though the html structure is not very straight forward, I think it's preferable.